tools/thinpool.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/sh
```

Comment:
This file should have a copyright header.

Reply:
Oh right. Will do.